src/callout.tsx

```diff
@@ -107,7 +111,11 @@ const Callout: React.FC<Props> = ({
                                 bleedRight
                                 Icon={IconCloseRegular}
                                 onPress={onClose}
-                                aria-label={closeButtonLabel ?? texts.closeButtonLabel}
+                                aria-label={
+                                    closeButtonLabel ||
+                                    texts.closeButtonLabel ||
+                                    translate(closeButtonLabelText, locale)
```

Comment:
Note that I'm using `||` instead of `??`. I think we can consider empty strings as missing